README.md

```diff
@@ -103,6 +101,14 @@ Or build and run the tests without scripts:
 2. Run the e2e tests:
 The `e2e-appstudio` command is the root command that executes all test functionality. To obtain all available flags for the binary please use `--help` flags. All ginkgo flags and go tests are available in `e2e-appstudio` binary.
 
+Some tests could require you to have specific container image repo's created (if you're using your own container image org/user account (`QUAY_E2E_ORGANIZATION`) or your own GitHub organization (`MY_GITHUB_ORG`) 
+In that case, before you run the test, make sure you have created
+* `test-images` repo in quay.io, i.e. `quay.io/<QUAY_E2E_ORGANIZATION>/test-images` and make it **public**
+  * also make sure that the docker config credentials (encoded in the value of `QUAY_TOKEN` environment variable) contain a correct credentials (with proper permissions) that will be used for pushing images to that repo
```

Comment:
Do we need this note as well: `make sure the robot account you're using to push to test-images repo has the required permissions set for that "test-images" repo.`? or it's too obvious?

Reply:
thanks, I rephrased that, but not sure if it didn't end up being totally confusing 🙈 please check and in case it's a disaster, please add a suggestion.